packages/amplify-category-geo/src/service-walkthroughs/placeIndexWalkthrough.ts

```diff
@@ -50,7 +50,7 @@ export const placeIndexNameWalkthrough = async (context: any): Promise<Partial<P
         const [shortId] = uuid().split('-');
         const indexNameInput = await prompter.input(
             'Provide a name for the location search index (place index):',
-            { validate: alphanumeric(), initial: `placeindex${shortId}` }
+            { validate: alphanumeric(), initial: `placeIndex${shortId}` }
```

Comment:
Just checking that this won't impact existing customers already having a placeIndex resource with `placeindex` in its name.

Reply:
From what I read in the docs for geo it seems like it would not break customers. I can follow up with someone from geo.